docs/sigs/Marketing.md

```diff
@@ -6,25 +6,38 @@ title: "Marketing SIG"
 The Marketing team is responsible for AlmaLinux marketing efforts, and we use the [Marketing](https://chat.almalinux.org/almalinux/channels/marketing)
 chat channel for communication on the AlmaLinux chat server.
 
+Meetings are every other Friday on Google Meet at 3pm UTC. Join the Marketing channel on [chat.almalinux.org](chat.almalinux.org) and ask benny for the meeting invite!
 
 ## Deliverables
 
 * Event plans
 * Outreach efforts
 * End user "help me convince my team to use AlmaLinux" support
 
-
 ## Help wanted
 
 * Event staffers
 * Content writers (especially for the blog and social media)
 * Website maintainers
 * Graphic designers
 
-
 ## SIG members
 
-* [benny Vasquez](mailto:benny@almalinux.org) - Temporary team lead
+(titles are in the context of the Marketing SIG - these folks might also operate in other capacities elsewhere at AlmaLinux)
+
+* [benny Vasquez](mailto:benny@almalinux.org) - Temporary committee lead
 * [Jack Aboutboul](mailto:jack@almalinux.org) - Community Manager/PR
 * [Jonathan Wright](mailto:jonathan@almalinux.org) - Events 
 * [Cody Robertson](mailto:crobertson@almalinux.org) - General Volunteer 
+
+We also have a variety of folks who help us with PR, social media, and website maintenance through submissions to chat or just PRs to the [website repo](https://github.com/AlmaLinux/almalinux.org).
+
+## Social Media Accounts
+
+We currently have a number of social media accounts that are managed by the SIG members.
+
+* [Mastodon (on fosstodon.org)](https://fosstodon.org/@almalinux) - password in Bitwarden
+* [Twitter (x)](https://twitter.com/almalinux) - password in Bitwarden
+* [Facebook](https://www.facebook.com/AlmaLinux/) - still tracking down admin rights
+* [LinkedIn](https://www.linkedin.com/company/almalinuxos/) - benny and Jack have 'owner' rights
```

Comment:
I see several others listed as well.  Two other "super admins" besides you and jack and I have content admin access.  Not sure if we want to list it all or what but we should at least be thorough on listing the super admins IMO.

Reply:
This is mostly meant to help in the event that someone is looking for admin access, rather than provide a comprehensive list. Thus far it's been a fun game of hot-potato, so this is just keeping at least SOME list somewhere.

Reply:
Ok I was wondering if that was the case.  In that case, LGTM.